_events/sr2021/league-2.md

```diff
@@ -1,10 +1,22 @@
 ---
 title: Competition League Session 2
-date: 2021-02-06
-time_tbc: true
+date: 2021-02-06 12:00:00 Europe/London
 layout: event
 type: competition
-location: Student Robotics' YouTube Channel
+location: <a href="https://www.youtube.com/watch?v=RwW5Oz30gbE">Student Robotics' YouTube Channel</a>
 ---
 
 This is the second league round of this year's competition and we'll get to see what lessons teams have learnt from their first time in the arena! Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 8pm GMT on Friday 5th: [Submit your code](https://studentrobotics.org/code-submitter/)
```

Comment:
Wonder if we can reword this to be for a target audience of the general public, while still useful to teams that find the  the page?

```suggestion
Teams will have to submit their code to us by 8pm GMT on Friday 5th using our [code submitter](https://studentrobotics.org/code-submitter/)
```